### PR TITLE
TrendMicroDDA - Add encoding to the url and an UT

### PR DESCRIPTION
**Pull request by rshunim**
## Status
- [x] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes:[ link to the issue](https://jira-hq.paloaltonetworks.local/browse/XSUP-24781)

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/TrendMicroDDA/Integrations/TrendMicroDDA/TrendMicroDDA_test.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import pytest
+import demistomock as demisto
+
+
+@pytest.mark.parametrize('sample_url', ["https%3A%2F%2Fwww.youtube.com%2Fwatch%3Fv%3DgrBFMP3HDZA"])
+def test_simple_upload_sample_url(mocker, requests_mock, sample_url):
```
Comment (yuvalbenshalom) [inline Packs/TrendMicroDDA/Integrations/TrendMicroDDA/TrendMicroDDA_test.py:6]: please just add explanation to what is being tested here.
also, for 1 test, there is no need to parametrize.

File: Packs/TrendMicroDDA/ReleaseNotes/1_0_13.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+
+#### Integrations
+
+##### Trend Micro Deep Discovery Analyzer (Beta)
+
+- Fixed an issue where ***simple-upload-sample-url*** command was failing due to unexpected url.
```
Comment (yuvalbenshalom) [inline Packs/TrendMicroDDA/ReleaseNotes/1_0_13.md:6]:
```suggestion
- Fixed an issue where ***simple-upload-sample-url*** command was failed to run on legitimate URL input.
```

**Review by yuvalbenshalom — CHANGES_REQUESTED** [review]
see my comment

File: Packs/TrendMicroDDA/Integrations/TrendMicroDDA/TrendMicroDDA_test.py (hunk 1 above)
Comment (rshunim) [reply Packs/TrendMicroDDA/Integrations/TrendMicroDDA/TrendMicroDDA_test.py]: Done

**Comment by rshunim** [comment]
ForceMerge is needed here since there was no test file, I created one and added UT, but still not enough coverage

**Review by yuvalbenshalom — APPROVED** [review]

**Pull request merged**